src/core/ext/filters/client_channel/lb_policy_factory.h

```diff
@@ -66,9 +66,18 @@ typedef struct grpc_lb_addresses {
 grpc_lb_addresses *grpc_lb_addresses_create(
     size_t num_addresses, const grpc_lb_user_data_vtable *user_data_vtable);
 
+/** Deep copy the address from \a src to \a dest. */
+void grpc_lb_address_copy(grpc_lb_address* dest, const grpc_lb_address* src,  
+                          const grpc_lb_user_data_vtable* user_data_vtable);
+
 /** Creates a copy of \a addresses. */
 grpc_lb_addresses *grpc_lb_addresses_copy(const grpc_lb_addresses *addresses);
 
+/** Extract the backend addresses and return them in a new \a 
+    grpc_lb_addresses. */
+grpc_lb_addresses *grpc_lb_addresses_extract_backends(
```

Comment:
The problem of changing the existing function is that `[grpc_arg_pointer_vtable]`https://github.com/grpc/grpc/blob/b48f0946a145893cb28117ef4f162c0a5050e577/include/grpc/impl/codegen/grpc_types.h#L87 should also change. It looks like I shouldn't force the siblings of `lb_addresses_arg_vtable` to take this change. Should I roll back (this rollback)?